src/GafferSceneUI/StandardLightVisualiser.cpp

```diff
@@ -454,16 +455,27 @@ Visualisations StandardLightVisualiser::visualise( const IECore::InternedString
 	}
 	else if( type && type->readable() == "quad" )
 	{
-		if( drawShaded )
+		ConstV2fDataPtr sizeData = Metadata::value<V2fData>( metadataTarget, "defaultSize" );
+		const V2f size = sizeData ? sizeData->readable() : V2f( 1.0f );
```

Comment:
Sorry, one more comment. I don't understand why we've introduced `defaultSize`, nor why it would default to 1 when previously it was 2. I think this is only going to confuse matters when we want to support genuine parameters for the quad size, so can we just remove it?

Reply:
Removed in squash https://github.com/GafferHQ/gaffer/pull/3533/commits/93f35d8e86050b362b4a55629a6851a019bdd2d7.